src/common/base/status.h

```diff
@@ -147,6 +147,12 @@ inline Status StatusAdapter<px::statuspb::Status>(const px::statuspb::Status& s)
 
 }  // namespace px
 
+#define PX_RETURN_STATUS_OK_IF_OK(__materialized_status, __status_gen) \
```

Comment:
imo this macro is more confusing than it's worth to save a couple lines, but if you really want to use it than you should wrap it in a do while false like the others

Reply:
There was a long standing TODO in the previous version of `linux_headers.cc` and I agreed fully with the insight behind that, however, given that this macro isn't used elsewhere, would be a good middle ground to just re-locate the macro into `linux_headers.cc`? We can pull it into `base` later if we want to.

Reply:
Also, realized we don't want the braces for this macro.